packages/flutter/lib/src/material/theme.dart

```diff
@@ -104,7 +104,7 @@ class Theme extends StatelessWidget {
   /// [BuildContext] that is "under" the [Theme]:
   ///
   /// ```dart
-  /// @override
+  /// @overrideun
```

Comment:
typo?

Reply:
Yes. Matches my predilection for ^u^n in emacs.